### ISSUE TITLE
/transfer <address> <port> not working

### ISSUE THREAD

**Issue by MasterLuke117** [PetteriM1/SynapseTools#2]
is 
```
/transfer <address> <port> <optional: player> 
```
supposed to still work when using SynapseTools?  

or does it override it to make /srv <description> work?

TransferCommand v1.0 is installed, 
running latest nukkit, synapses and nemisys (official) 

it makes no sense, some servers /transfer returns as:

```
/transfer <target> 
```
and some return as
```
/transfer <address> <port> <optional: player>
```
all servers have the same plugins and config settings.

**Comment by PetteriM1** [comment]
It depends of the order how server loads plugins on startup. I can try to find a way to load this plugin always before transfer so the commands gets overwritten in correct order.

**Comment by MasterLuke117** [comment]
Fixed!  SynapseTools-1.0.2 did it,  Thanks!
